src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -797,6 +797,17 @@ function fs.complete(sPath, sLocation, bIncludeFiles, bIncludeDirs)
     return tEmpty
 end
 
+function fs.isMountPoint(sPath)
```

Comment:
"Mount point" feels like a bit of an odd name. It's obviously technically correct, but but not really something CC uses elsewhere. I don't know if `isDriveRoot` is any clearer (given `getDrive` is a thing)?

Can you add some docs to `doc/stub/fs.lua` too? Hopefully one day we'll be smart enough to pull them in from the bios.

Reply:
I feel like `isDriveRoot` would be confusing, it sounds like it should only return true for `/`. Might be helpful to consider that disks call this `getMounthPath`, so `isMountPath` might also work.

Reply:
how about `isMountRoot`?